package.json

```diff
@@ -124,6 +125,7 @@
     "pngjs": "^3.3.1",
     "pretty-hrtime": "^1.0.3",
     "promisify-event": "^1.0.0",
+    "prompts": "^2.4.2",
```

Comment:
Considering I have a default dashboard token set om ,y machine but the sending mode is 'switched-off'. Should we include the info about switched-off sending mode into `testcafe dashboard` command output in this case? 
![image](https://user-images.githubusercontent.com/12034551/166479697-25b8b053-d282-49d5-a24b-35d8c7ea0587.png)
@kirovboris @titerman 
I think we should.

Reply:
I'd not overload the dialog text and enable sending reports if the authorization was successful.

Reply:
I agree with both @kirovboris and @helen-dikareva. We can probably compromise. Can we display the following message when the person changes the default token but the reporter is disabled?

```
    TOKEN_UPDATED:                            'The default TestCafe Dashboard token has been updated. Enable the Dashboard reporter to start uploading test data.',
```

Reply:
Ok.

Reply:
Ok, let's stick with this behaviour.